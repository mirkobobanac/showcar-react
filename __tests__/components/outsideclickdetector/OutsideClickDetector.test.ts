import initStoryshots from '@storybook/addon-storyshots'

initStoryshots({
  storyKindRegex: /\/OutsideClickDetector$/
})

// Additional OutsideClickDetector tests
