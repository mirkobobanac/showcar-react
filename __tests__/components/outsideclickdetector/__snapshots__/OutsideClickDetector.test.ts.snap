// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Core|Utilities/OutsideClickDetector Click Detection 1`] = `
<div
  style={
    Object {
      "backgroundColor": "pink",
      "margin": "0 auto",
      "maxWidth": "400px",
      "padding": "2em",
    }
  }
>
  <div
    style={
      Object {
        "backgroundColor": "wheat",
        "margin": "0 auto",
        "padding": "2em",
        "textAlign": "center",
        "width": "200px",
      }
    }
  >
    <div>
      I know when someone clicks outside this box...
    </div>
  </div>
</div>
`;

exports[`Storyshots Core|Utilities/OutsideClickDetector usage 1`] = `
<div>
  <div
    style={Object {}}
  >
    <div />
  </div>
  <div>
    <div
      style={
        Object {
          "WebkitFontSmoothing": "antialiased",
          "backgroundColor": "#fff",
          "border": "1px solid #eee",
          "borderRadius": "2px",
          "boxShadow": "0px 2px 3px rgba(0, 0, 0, 0.05)",
          "color": "#444",
          "fontFamily": "-apple-system, \\".SFNSText-Regular\\", \\"San Francisco\\", BlinkMacSystemFont, \\"Segoe UI\\", \\"Roboto\\", \\"Oxygen\\", \\"Ubuntu\\", \\"Cantarell\\", \\"Fira Sans\\", \\"Droid Sans\\", \\"Helvetica Neue\\", \\"Lucida Grande\\", \\"Arial\\", sans-serif",
          "fontSize": "15px",
          "fontWeight": 300,
          "lineHeight": 1.45,
          "marginBottom": "20px",
          "marginTop": "20px",
          "padding": "20px 40px 40px",
        }
      }
    >
      
      <div>
        <h1
          style={
            Object {
              "borderBottom": "1px solid #EEE",
              "fontSize": "25px",
              "margin": "20px 0 0 0",
              "padding": "0 0 5px 0",
            }
          }
        >
          Prop Types
        </h1>
        <div>
          <h2
            style={
              Object {
                "margin": "20px 0 0 0",
              }
            }
          >
            "
            OutsideClickDetector
            " Component
          </h2>
          <small>
            No propTypes defined!
          </small>
        </div>
      </div>
    </div>
  </div>
</div>
`;
